src/components/Badge/Badge.tsx

```
@@ -0,0 +1,146 @@
+  const baseClasses =
+    'inline-flex items-center justify-center rounded-xs font-medium gap-1';
+
+  const baseClassesIcon = 
+    'flex items-center';
+  if(variant == 'notification'){
+    return (
+      <div
+        className={`${baseClasses} ${variantClasses} ${sizeClasses} ${className}`}
+        {...props}
+      >
+        <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```

ESLint

[error] Replace `·width="24"·height="25"·viewBox="0·0·24·25"·fill="none"·xmlns="http://www.w3.org/2000/svg"` with `⏎··········width="24"⏎··········height="25"⏎··········viewBox="0·0·24·25"⏎··········fill="none"⏎··········xmlns="http://www.w3.org/2000/svg"⏎········`

(prettier/prettier)